server/src/main/java/org/opensearch/extensions/ExtensionStringResponse.java

```diff
@@ -17,7 +17,7 @@
 /**
  * Generic string response indicating the status of some previous request sent to the SDK
  *
- * @opensearch.internal
+ * @opensearch.api
```

Comment:
I was going to write "Do we expect users to use this class" and then realized the answer is yes, so +1 on this.